primers.go

```diff
@@ -103,3 +104,80 @@ func MeltingTemp(sequence string) float64 {
 	meltingTemp, _, _ := SantaLucia(sequence, primerConcentration, saltConcentration, magnesiumConcentration)
 	return meltingTemp
 }
+
+/******************************************************************************
+Start of the De Bruijn stuff
+******************************************************************************/
+
+// DeBruijn generates a DNA DeBruijn sequence with alphabet ATGC. DeBruijn sequences are basically a string with all unique substrings of an alphabet represented exactly once.
+// https://en.wikipedia.org/wiki/De_Bruijn_sequence
+// https://rosettacode.org/wiki/De_Bruijn_sequences#Go
+// Pulled and adapted from here
+func DeBruijn(n int) string {
+	k := 4
```

Comment:
what is k?

Reply:
And why is it 4? Is this the length of the `alphabet` variable below?

Reply:
```suggestion
	alphabetSize := alphabet.length
```

Reply:
Yep, it was based on length of alphabet.

Reply:
Fixed